### PR TITLE
Update performance-modeling.md

### PR DESCRIPTION
**Pull request by nkaputnik**
We have updated the calculated fields section to now reflect the new CDS calculated element capabilities. @renejeglinsky we do not know how to properly link other Capire pages within the markup, so I pasted two absolute links in the text. Can you please replace them with the proper internal link syntax?

### CODE REVIEW

File: advanced/performance-modeling.md [hunk 1]
```diff
@@ -277,8 +277,8 @@ Typical examples of calculated fields are:
 The following steps show you which option takes precedence over another. Use options one/two as the preferred way and three/four as fallback.
 
 1. Do the calculation on the UI with help of field controls or dedicated custom controls. This applies to all kinds of **String concatenation** and **Formatting**.
-2. Pre-calculate on *write* with help of an event handler.
-3. Some Calculated Fields are dynamic in nature. Do those calculations on the database layer. For example _kanban_ (scheduling system for lean manufacturing), there you typically have dynamic live calculations.
+2. Pre-calculate using CDS [on write](https://pages.github.tools.sap/cap/docs/cds/cdl#on-write) calculated fields.
```
Comment (renejeglinsky) [inline advanced/performance-modeling.md:280]:
```suggestion
2. Pre-calculate using CDS [on write](../cds/cdl#on-write) calculated fields.
```

File: advanced/performance-modeling.md [hunk 2]
```diff
@@ -277,8 +277,8 @@ Typical examples of calculated fields are:
...
+3. Some Calculated Fields are dynamic in nature. If possible, calculate them using CDS [on read](https://pages.github.tools.sap/cap/docs/cds/cdl#on-read) calculated fields.
```
Comment (renejeglinsky) [inline advanced/performance-modeling.md:281]:
```suggestion
3. Some Calculated Fields are dynamic in nature. If possible, calculate them using CDS [on read](../cds/cdl#on-read) calculated fields. 
```

**Review by renejeglinsky — COMMENTED** [review]
adapted links

File: advanced/performance-modeling.md [hunk 3]
```diff
@@ -277,8 +277,8 @@ Typical examples of calculated fields are:
 The following steps show you which option takes precedence over another. Use options one/two as the preferred way and three/four as fallback.
 
 1. Do the calculation on the UI with help of field controls or dedicated custom controls. This applies to all kinds of **String concatenation** and **Formatting**.
-2. Pre-calculate on *write* with help of an event handler.
-3. Some Calculated Fields are dynamic in nature. Do those calculations on the database layer. For example _kanban_ (scheduling system for lean manufacturing), there you typically have dynamic live calculations.
+2. Pre-calculate using CDS [on write](../cds/cdl#on-write) calculated fields.
+3. Some Calculated Fields are dynamic in nature. If possible, calculate them using CDS [on read](../cds/cdl#on-read) calculated fields.
```
Comment (renejeglinsky) [inline advanced/performance-modeling.md:281]: That "suffix" `calculated fields` in both sentences, is that needed? Feels redundant when reading.

Comment (nkaputnik) [reply advanced/performance-modeling.md]: I changed the wording to be less redundant

**Pull request merged**